DB/projects.json

```
@@ -5772,3 +5772,28 @@
     ]
   }
 ]
```

add `,` after `}` & remove `]`
```suggestion
  },

```